core.cpp

```diff
@@ -88,13 +88,18 @@ void runStateClass::SetManual(bool val, int8_t zone)
 
 #ifdef ARDUINO
 uint8_t ZoneToIOMap[] = {22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37};
+#elif defined(GREENIQ)
+uint8_t ZoneToIOMap[] = {5,7,0,1,2,3,4};
+#define NW_LED			11
+#define LIGHT			6
 #else
 uint8_t ZoneToIOMap[] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15};
+#endif
 #define SR_CLK_PIN  7
 #define SR_NOE_PIN  0
 #define SR_DAT_PIN  2
 #define SR_LAT_PIN  3
-#endif
+
```

Comment:
Please define these in your section explicitly. If you move then #endif it will define in ARDUINO when it should not.